.prettierignore

```diff
@@ -0,0 +1 @@
+kuma/javascript/src/flow-typed/
```

Comment:
Running `npm run format` locally leaves the js files in flow-typed/ alone. But, for whatever reason, running `docker-compose exec web npm run format` runs prettier over those files. So I've added this config file to exclude that directory from formatting.

Reply:
Can't this be added to the existing `.prettierrc` file?

Reply:
Not as far as I can tell.

Reply:
Perhaps what you *can* do is specify the location of the file that tells with files to ignore :)